src/components/books/Book.jsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
 
 import styles from './Book.module.scss';
+import { deleteBook } from '../../redux/books/books';
 
 const Book = ({ book }) => {
-  const {
-    author, title, category, progress,
-  } = book;
-  const { currentChapter } = progress;
+  // eslint-disable-next-line
+  const { item_id, author, title, category, completed } = book;
+  const dispatch = useDispatch();
+  const handleRemoveBtnClick = () => {
+    dispatch(deleteBook(item_id));
+  };
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, please avoid disabling linter rules! Linters help the code stay consistent, and clean and ensure the use of best practices. I think that the rule you need to solve for this one is the `"camelcase"` one. You could easily solve it by setting an alias to the object key, like this: 🤓 🔧 

```suggestion

  const { item_id: itemId, author, title, category, completed } = book;
  const dispatch = useDispatch();
  const handleRemoveBtnClick = () => {
    dispatch(deleteBook(itemId));
  };
```

Reply:
![image](https://user-images.githubusercontent.com/55840999/214172172-c8838674-ded8-45da-99a1-bdf8a62d32d9.png)
Thanks for your comment. But I wanted to use the same **item_id** as [book api](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739).  Because I have to work with it in my next implementation.
@anagudelogu What do you suggest me to do now with it?